src/IssueViz.Security/Taint/TaintIssuesBindingMonitor.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using SonarLint.VisualStudio.Core.Binding;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.Taint
+{
+    internal interface ITaintIssuesBindingMonitor : IDisposable
+    {
+        void ListenToBindingChanges();
+    }
+
+    [Export(typeof(ITaintIssuesBindingMonitor))]
+    internal sealed class TaintIssuesBindingMonitor : ITaintIssuesBindingMonitor
```

Comment:
It would be useful to have a doc comment explaining the purpose of the class e.g. `Fetches the taint issues from connected server when the binding changes`